pyomo/dataportal/plugins/xml_table.py

```diff
@@ -9,14 +9,29 @@
 #  ___________________________________________________________________________
 
 import os.path
-try:
-    import lxml.etree.ElementTree as ET
-except:
-    import xml.etree.ElementTree as ET
-
+from pyomo.common.dependencies import attempt_import
 from pyomo.dataportal.factory import DataManagerFactory
 from pyomo.dataportal import TableData
 
+def _xml_importer():
+    try:
+        from lxml import etree
+        return etree
+    except ImportError:
+        pass
+
+    try:
+        # Python 2.5+
```

Comment:
Did you mean to type Python 2.5 here or is this a typo?

Reply:
Yes.  I meant Pyomo 2.5 (that syntax works as far back as python 2.5, but no farther).  There is reference documentation that also includes imports for 2.4 and earlier, but I intentionally did not include those imports here.